Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -537,6 +546,8 @@ bool FullscreenManager::willEnterFullscreen(Element& element)
             containingBlockBeforeStyleResolution = renderer->containingBlock();
 
         ancestor->setFullscreenFlag(true);
+        if (auto* frameElement = dynamicDowncast<HTMLIFrameElement>(ancestor.get()))
+            frameElement->setIFrameFullscreenFlag(false);
```

Comment:
I can't remember why I set the flag to false in this specific case, it's not in the spec: https://fullscreen.spec.whatwg.org/#dom-element-requestfullscreen

Might be possible to remove it.

Reply:
I'd add a FIXME.